src/Units/MoveTypes/Lander.java

```diff
@@ -0,0 +1,14 @@
+package Units.MoveTypes;
+
+public class Lander extends MoveType
```

Comment:
`Lander` sounds like a unit name; I'd rather not have it also be the move type name. We should standardize how we name move types for clarity.